core/core_integrations/facebook.js

```diff
@@ -1,170 +1,170 @@
-		}
+    fb({email: this.config.username, password: this.config.password}, function (err, api) {
+        if (err) {
+            console.error(err);
+            process.exit(-1);
+        }
 
-		var options = {
-			listenEvents: true
-		};
-		if (!console.isDebug()) {
-			options.logLevel = 'silent';
-		}
-		api.setOptions(options);
-		platformApi = api;
+        var options = {
+            listenEvents: true
+        };
+        if (!console.isDebug()) {
+            options.logLevel = 'silent';
+        }
+        api.setOptions(options);
+        platformApi = api;
 
-		platform = shim.createPlatformModule({
-			commandPrefix: exports.config.commandPrefix,
-			sendMessage: function(message, thread) {
-				stopTyping();
-				api.sendMessage({body:message}, thread);
-			},
-			sendUrl: function(url, thread) {
-				stopTyping();
-				api.sendMessage({body: url, url: url}, thread);
-			},
-			sendImage: function(type, image, description, thread) {
-				stopTyping();
-				switch (type) {
-					case "url":
-						api.sendMessage({body: description, url: image}, thread, function(err, messageInfo) {
-							if (err) {
-								api.sendMessage(description + " " + image, thread);
-							}
-						});
-						break;
-					case "file":
-						api.sendMessage({body: description, attachment: fs.createReadStream(image)}, thread);
-						break;
-					default:
-						api.sendMessage(description, thread);
-						api.sendMessage(image, thread);
-						break;
-				}
-			},
-			sendFile: this.sendImage,
-			sendTyping: function(thread) {
-				stopTyping();
-				api.sendTypingIndicator(thread, function(err, end) {
-					if (!err) {
-						endTyping = end;
-					}
-				});
-			},
-			setTitle: function(title, thread) {
-				stopTyping();
-				api.setTitle(title, thread);
-			},
-			getUsers: function(thread) {
-				return threadInfo[thread];
-			}
-		});
+        platform = shim.createPlatformModule({
+            commandPrefix: exports.config.commandPrefix,
+            sendMessage: function(message, thread) {
+                stopTyping();
+                api.sendMessage({body:message}, thread);
+            },
+            sendUrl: function(url, thread) {
+                stopTyping();
+                api.sendMessage({body: url, url: url}, thread);
+            },
+            sendImage: function(type, image, description, thread) {
+                stopTyping();
+                switch (type) {
+                    case 'url':
+                        api.sendMessage({body: description, url: image}, thread, function(err) {
+                            if (err) {
+                                api.sendMessage(description + " " + image, thread);
+                            }
+                        });
+                        break;
+                    case 'file':
+                        api.sendMessage({body: description, attachment: fs.createReadStream(image)}, thread);
+                        break;
+                    default:
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 space characters but found 20.](https://www.codacy.com/app/theprojectshop/Kassy/file/2589168481/issues/source?bid=3282987&fileBranchId=3291254#l92)